proto/pbconfigentry/config_entry.proto

```diff
@@ -602,6 +602,7 @@ message Status {
 // name=Structs
 message Condition {
   string Type = 1;
+  // mog: func-from=conditionStatusFromStructs func-to=conditionStatusToStructs
```

Comment:
Why does this not seem to be working, do I need to regenerate something?

Reply:
Needed to reinstall Xcode developer tools to get `make tools` and then `make proto` to run successfully.